packages/back-end/types/organization.d.ts

```diff
@@ -76,12 +76,16 @@ export interface PendingMember extends MemberRoleWithProjects {
 export interface Member extends MemberRoleWithProjects {
   id: string;
   dateCreated?: Date;
+  externalId?: string;
+  managedByIdp?: boolean;
 }
 
 export interface ExpandedMember extends Member {
   email: string;
   name: string;
   verified: boolean;
+  managedByIdp?: boolean;
+  externalId?: string;
```

Comment:
Don't need this here since it extends `Member` and will already pick up these fields